exporter/collector/README.md

```diff
@@ -43,19 +43,15 @@ These instructions are to get you up and running quickly with the GCP exporter i
         loglevel: debug
     processors:
       memory_limiter:
-      batch:
-        # Google Cloud Monitoring limits batches to 200 metric points.
-        send_batch_max_size: 200
-        send_batch_size: 200
     service:
       pipelines:
         traces:
           receivers: [otlp]
-          processors: [memory_limiter, batch]
```

Comment:
We only do batching for metrics, right?  Should we still recommend it for traces?

Reply:
Good point. I've changed this to only recommending it for traces.